### DISCUSSION TITLE
webhook

### DISCUSSION THREAD

**Discussion by kami0314** [nanomq/nanomq#454] [Ideas]
Are there plans to add Webhook features？

**Comment by JaylinYu** [comment]
webhook just like EMQX or a bash hook with more possibilities? https://github.com/nanomq/nanomq/discussions/297

**Reply by kami0314** [reply]
just like EMQX

**Comment by JaylinYu** [comment]
well, not a big deal. will do

**Reply by kami0314** [reply]
cool！When is this feature expected to be available？

**Reply by JaylinYu** [reply]
coming soon. should be in our next beta release if no any other mistakes

**Comment by JaylinYu** [comment]
it is already done in the master branch. we will release this feature in 0.7.0

**Reply by JaylinYu** [reply]
@kami0314  any feedback on our webhook?

**Reply by kami0314** [reply]
> 
Tested, no problem！

**Comment by JaylinYu** [comment]
webhook is officially released now
